man/systemd-battery-check.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
+
+<refentry id="systemd-battery-check" xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>systemd-battery-check</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>systemd-battery-check</refentrytitle>
+    <manvolnum>1</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>systemd-battery-check</refname>
+    <refpurpose>Checks battery level to see whether there's enough charge.</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <cmdsynopsis>
+      <command>systemd-battery-check</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+    </cmdsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><command>systemd-battery-check</command> is primarily used to check the battery level during the early boot
+         stage to determine whether there's sufficient battery power to carry on with the booting process.
+        However, it may also be useful in other situations where it's necessary to ensure
+        sufficient battery power. The tool returns success if the device is connected to an AC power source
```

Comment:
hmm, i think it'd drop the sentence starting with "However, …"

given it writes to /dev/console it's not really useful outside of the intended usecase.

it migh tmake sense to extend the systemd-ac-power tool to also be useful to check for low power state, but that's really a different discussion